test/arbitrage/BancorArbitrage.ts

```diff
@@ -1030,6 +1108,48 @@ describe('BancorArbitrage', () => {
             );
         });
 
+        it('should be able to arbitrage using Bancor V3', async () => {
```

Comment:
instead of a dedicated test, we can re-enable the ones we initially disabled

Reply:
Removed that test - the others are enabled.